.storybook/static/preview-head.js

```diff
@@ -375,3 +375,44 @@ function toggleNestedListSubmenu(event) {
         icon.classList = 'sap-icon--navigation-down-arrow';
     }
 }
+
+function collectionHas(a, b) { //helper function (see below)
+    for(var i = 0, len = a.length; i < len; i ++) {
+        if(a[i] == b) return true;
```

Comment:
What is `a[i]` here? 
Isn't it better to use `===` instead of `==`?
I think this line can be simplified like: `return a[i] === b;`

Reply:
`a` is the collection and `b` is an item that may or may not be in that collection and `i` is the index of the collection, `a[i]` means the item at index `i` in collection `a`

Reply:
Also I just realized this function is unused so it can be deleted

Reply:
deleted the function